### PR TITLE
Update SetPedCanPlayAmbientIdles documentation

### PR DESCRIPTION
**Pull request by coalaura**
I was playing around with a variety of natives to try to disable the idle animation on player peds. This native did the trick, after experimenting with it, it turns out it only works if p1 and p2 are true. Later noticed it was not syncing between clients, so I had to have a loop in my script, calling it for each active player ped for it to work.

### CODE REVIEW

**Comment by freedy69** [comment]
lol thank you for this i never understood how this native worked

File: TASK/SetPedCanPlayAmbientIdles.md [hunk 1]
```diff
@@ -9,15 +9,16 @@ aliases: ["0x8FD89A6240813FD0"]
...
+while true do
+    SetPedCanPlayAmbientIdles(PlayerPedId(), true, true)
+    Citizen.Wait(0)
+end
 ```
 
 ## Parameters
 * **ped**: 
 * **p1**: 
 * **p2**:
```
Comment (PsychoShock) [inline TASK/SetPedCanPlayAmbientIdles.md:24]: * **ped**: The ped index.
* **blockidleclips**: If true, blocks future idle clips from playing.
* **removeclip**: If true, will remove any current clip playing (will also block).


This is the missing params :D

Comment (coalaura) [reply TASK/SetPedCanPlayAmbientIdles.md:24]: thank you! I'll update my PR

**Comment by coalaura** [comment]
did not mean to delete the branch

File: TASK/SetPedCanPlayAmbientIdles.md (hunk 1 above)
Comment (coalaura) [reply TASK/SetPedCanPlayAmbientIdles.md]: merged with master

**Comment by coalaura** [comment]
Looking at it, i believe https://github.com/citizenfx/natives/commit/e478807fbc33fc940df2e35f7287614142fbbf8d is a better solution

**Pull request closed without merging**